draft-ietf-httpbis-priority.md

```diff
@@ -309,6 +309,29 @@ properties of an HTTP request it receives, the server is expected to control the
 cacheability or the applicability of the cached response, by using header fields
 that control the caching behavior (e.g., Cache-Control, Vary).
 
+# Signalling Initial Priority in Frames
+
+Endpoints might prefer to signal the initial priority of a response using a
+frame rather than a header field. This document specifies a new PRIORITY_UPDATE
+frame for HTTP/2 ({{!RFC7540}}) and HTTP/3 ({{!I-D.ietf-quic-http}}). It carries
+priority parameters and references the target of the prioritization based on a
+version-specific identifier; in HTTP/2 this is the Stream ID, in HTTP/3 this is
+either the Stream ID or Push ID.
```

Comment:
How about stating that endpoints that do not want to expose an end-to-end signal MAY omit the header field, but send the PRIORITY_UPDATE frame at an early moment assuming that it would reach the server as early as the request headers? Would could go on to explain that in such case, all the responses that the server would send would have the priority in the frame being applied.

The benefit of phrasing that way is that we can get rid of the last paragraph of this section.

Reply:
I've tried to incorporate this suggestion in https://github.com/httpwg/http-extensions/pull/1167/commits/f5ac92a045a3074e503c93d304a90799e387ab07 but found it trickier to write than I first thought it would be, so it ended up a little different in the end. Please take a look and add some further suggestions if I'm not hitting your mark.